static/sass/_snapcraft_p-navigation.scss

```diff
@@ -20,11 +20,15 @@
         // starts on the right outside of the screen
         --right: 110%;
 
-        &--wrapper {
+        .p-navigation__items-wrapper {
```

Comment:
Does this style need to be here (nested under `p-navigation__nav`)?

We already have styles for `p-navigation__items-wrapper` separately in line 79. Could the `position: relative` be moved there to keep all styles together?